portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java

```diff
@@ -109,11 +109,23 @@ public static Locale fromLanguageId(
 	}
 
 	public static Locale[] fromLanguageIds(List<String> languageIds) {
-		return _localeUtil._fromLanguageIds(languageIds);
+		return _localeUtil._fromLanguageIds(languageIds, true);
+	}
+
+	public static Locale[] fromLanguageIds(
```

Comment:
@marianoalvarosaiz You added two methods, but only one is going to be used, and there is only one usage of it. I am wondering if there is a way to avoid adding these two methods.